blockly-ws-search/workspace_search.js

```diff
@@ -302,13 +327,25 @@ class WorkspaceSearch {
    */
   open() {
     this.setVisible(true);
+    this.updateMarker_();
     this.textInput_.focus();
     if (this.searchText_) {
       this.search();
     }
     console.log("Open search bar");
   }
 
+  /**
+   * Mark the user's current position when opening the search bar.
+   */
+  updateMarker_() {
+    var marker = this.workspace_.getMarker(Blockly.navigation.MARKER_NAME);
+    if (this.workspace_.keyboardAccessibilityMode && !marker.getCurNode()) {
```

Comment:
Since `getMarker` can return `null`, should we add a check in the `if`? or is it sufficient that the workspace is in `keyboardAccessibilityMode`?

Reply:
We always create this marker in createDom for the workspace so it should be fine, but it won't hurt so I will add.